src/OpenTelemetry/Metrics/AggregatorStore.cs

```diff
@@ -81,6 +86,21 @@ internal sealed class AggregatorStore
             this.tagKeysInteresting = hs;
             this.tagsKeysInterestingCount = hs.Count;
         }
+
+        // If the switch is not set at all or if it's explicitly set to false
+        if (AppContext.TryGetSwitch("OTel.Dotnet.EmitMetricOverflowAttribute", out bool shouldEmitOverflowAttribute) == false ||
```

Comment:
nit: maybe move the logic to check if this feature is enabled/diabled to MeterProvider, and have it pass a simple bool to AggregatorStore.

Reply:
I have updated the code.